palantir/datasets/core.py

```diff
@@ -212,18 +212,18 @@ def abort(self) -> None:
         self.client.abort_transaction(self)
         self.status = TransactionStatus.ABORTED
 
-    def write(self, path: str, content: bytes) -> None:
+    def write(self, logical_path: str, content: bytes) -> None:
```

Comment:
I think we should just treat this as `path` externally, the implementation distinction should not be relevant here.